plugins/outputs/sql/sqlite_test.go

```diff
@@ -15,10 +15,8 @@ import (
 )
 
 func TestSqlite(t *testing.T) {
-	outDir := os.TempDir()
-	defer os.Remove(outDir)
-
-	dbfile := filepath.Join(outDir, "db")
+	dbfile := filepath.Join(os.TempDir(), "db")
+	defer os.Remove(dbfile)
```

Comment:
I think this will leave the temporary directory... So I would prefer to have the original code but change `defer os.Remove(outDir)`  to `defer os.RemoveAll(outDir)` to recursively remove the temporary directory...

Reply:
> defer os.RemoveAll(outDir)

This would remove everything in `/tmp` for example. I forgot this, but TempDir doesn't create a tempdir it gets your default temporary directory on your platform. So originally we were trying to delete everything in `/tmp` versus removing `/tmp/db`